src/main/java/org/broadinstitute/hellbender/utils/hdf5/HDF5File.java

```diff
@@ -1,1052 +0,0 @@
-package org.broadinstitute.hellbender.utils.hdf5;
```

Comment:
So do I understand it correctly, that all this code has been moved to the new hdf5 repo?